tests/misc/test_build/testcase.yaml

```diff
@@ -16,3 +16,8 @@ tests:
     build_only: true
     extra_configs:
       - CONFIG_GDB_SERVER=y
+  test_mcuboot_bootloader:
+    build_only: true
+    platform_whitelist: nrf52_pca10040
```

Comment:
Can we also include a platform that uses SW_VECTOR_RELAY?

Reply:
Will do